cosmos/operators/kubernetes.py

```diff
@@ -165,6 +165,9 @@ def __init__(self, on_warning_callback: Callable[..., Any] | None = None, **kwar
         if not on_warning_callback:
             super().__init__(**kwargs)
         else:
+            # Make the following code independent of arguments being defaulted or not.
```

Comment:
Why are we making this change only to the `DbtTestKubernetesOperator`? would it also make sense in the remaining operators?